frontend_lib/src/component/Input/TextInput.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+require('./TextInput.styl')
+
+// A thin-layer around <input type='text'> that sets the Tracim style and behaviour
```

Comment:
comment must use the required syntax:
INFO - developer - date - text

Reply:
It seems to me that a comment documenting the intent of a class should not need this format.
This is something we do not do in the backend for instance.

This format seems useful for wargings, fixmes, todos, or notes on something tricky in the code.